specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/examples/CreateOrUpdateASimpleGalleryImageVersionWithVHD.json

```diff
@@ -0,0 +1,215 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2020-09-30",
+    "galleryName": "myGalleryName",
+    "galleryImageName": "myGalleryImageName",
+    "galleryImageVersionName": "1.0.0",
+    "galleryImageVersion": {
+      "location": "West US",
+      "properties": {
+        "publishingProfile": {
+          "targetRegions": [
+            {
+              "name": "West US",
+              "regionalReplicaCount": 1,
+              "encryption": {
+                "osDiskImage": {
+                  "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myDiskEncryptionSet"
+                },
+                "dataDiskImages": [
+                  {
+                    "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myOtherDiskEncryptionSet",
+                    "lun": 1
+                  }
+                ]
+              }
+            },
+            {
+              "name": "East US",
+              "regionalReplicaCount": 2,
+              "storageAccountType": "Standard_ZRS"
+            }
+          ]
+        },
+        "storageProfile": {
+          "source": {
+            "id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Compute/vhd/{vhdName}",
+            "uri": "http://mystorageaccount.core.windows.net/mycontainer/myblob?sv=2015-04-05&st=2015-04-29T22%3A18%3A26Z&se=2055-04-30T02%3A23%3A26Z&sr=b&sp=r&sip=168.1.5.60-168.1.5.70&spr=https&sig=tesT"
```

Comment:
also, this one would just be the vhd, not the sas

Reply:
ex, https://gallerysourcencus.blob.core.windows.net/myvhds/Windows-Server-2012-R2-20171216-en.us-128GB.vhd